api/v1beta1/experiment_types.go

```diff
@@ -117,6 +117,10 @@ type Metric struct {
 	Port intstr.IntOrString `json:"port,omitempty"`
 	// URL path component used to collect the metric value from an endpoint (used as a prefix for the Prometheus API)
 	Path string `json:"path,omitempty"`
+	// URL to query for fetching metrics.
```

Comment:
Do we want to just remove these from `v1beta1`?

Reply:
(that didn't highlight properly, by "these" I meant all the other URL like fields)

Reply:
Yeah we can, wanted to try to keep the scope of this minimal to make it easier to review :)